delta/sdk/src/main/resources/contexts/bulk-operation.json

```diff
@@ -0,0 +1,10 @@
+{
+  "@context": {
+    "_total": "https://bluebrain.github.io/nexus/vocabulary/total",
```

Comment:
Not sure if we need the total, we have in the search results because, there is pagination